tests/unittests/test_number_transform.py

```diff
@@ -1,25 +1,75 @@
 import unittest
```

Comment:
I can't find test cases for the function transform_sheet_decimal_data

Reply:
> Add unittest of the function `transform_sheet_decimal_data`

I have added test cases for `transform_sheet_number_data` which calls `transform_sheet_decimal_data` function.